src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -14,6 +13,8 @@ import '@webex/components/dist/css/webex-components.css';
  *
  * @param {string} props.meetingDestination  ID of the virtual meeting location
  * @param {string} props.accessToken        access token to create the webex instance with
+ * @param {string} props.className          Custom CSS class to apply
```

Comment:
[props.className] in square brackets to indicate it is optional. Same for style.